nilearn/decoding/space_net.py

```diff
@@ -621,17 +692,19 @@ class BaseSpaceNet(LinearRegression, CacheMixin):
         Each pair is the list of indices for the train and test samples
         for the corresponding fold.
 
-    `cv_scores_` : ndarray, shape (n_folds, n_alphas) or (n_l1_ratios, n_folds, n_alphas)
+    `cv_scores_` : ndarray, shape (n_folds, n_alphas)
```

Comment:
You need to explicitly break the line with a "\\" character, otherwise the rendering doc will only have the first line in bold and the second half will render in the description.

Reply:
Was not aware of that. Thanks.

Reply:
I fixed all those line breaks I had added.

Unrelated: when looking at the doc in html after building locally, it seems that the first line in bold is not a systematic thing for all arguments, no?